spring-kafka-docs/src/main/antora/modules/ROOT/pages/whats-new.adoc

```diff
@@ -83,3 +83,26 @@ See xref:kafka/seek.adoc#seek[Seek API Docs] for more details.
 For Spring Boot applications which define an application name, this name is now used
 as a default prefix for auto-generated client IDs for certain client types.
 See xref:kafka/connecting.adoc#default-client-id-prefixes[Default client ID prefixes] for more details.
+
+[[get-listener-containers-matching]]
+== Enhanced Retrieval of MessageListenerContainers
+
+Introduced in version 3.2, the `ListenerContainerRegistry` provides advanced methods for dynamically filtering `MessageListenerContainer` instances, enhancing management capabilities.
```

Comment:
This file is fully specific to particular version we mention in the beginning of the file.
More over the content of the file is just a list of topics with links to details.
You do show enough in the `kafkalistener-lifecycle.adoc`, but that all must not be duplicated in `What's New` file.

Reply:
True, thank you for pointing out!